core/sts-3.3.md

```diff
@@ -274,7 +311,7 @@ server may be offered at a different domain name, for example a subdomain.
 
 It's possible for an attacker to strip the STS `port` value from an initial connection
 established via an insecure connection, before the policy has been cached by the client.
-This represents a Bootstrap MITM (man-in-the-middle) vulnerability.
+This represents a bootstrap MITM (man-in-the-middle) vulnerability.
```

Comment:
Nit: Most of the literature I've seen doesn't capitalise the minor words in the initialism, so I'd expect to see `MitM`(e.g. see http://ieeexplore.ieee.org/document/7546529/?reload=true)